stdlib/2/xmlrpclib.pyi

```diff
@@ -0,0 +1,193 @@
+    def end_base64(self, data: str) -> None: ...
+    def end_dateTime(self, data: str) -> None: ...
+    def end_value(self, data: str) -> None: ...
+    def end_params(self, data: str) -> None: ...
+    def end_fault(self, data: str) -> None: ...
+    def end_methodName(self, data: str) -> None: ...
+
+class _MultiCallMethod: ...
+class MultiCallIterator: ...
+
+class MultiCall:
+    def __init__(self, server: ServerProxy) -> None: ...
+    def __getattr__(self, name: str) -> _MultiCallMethod: ...
+    def __call__(self) -> MultiCallIterator: ...
+
+def getparser(use_datetime: bool = ...) -> Tuple[Union[ExpatParser, SlowParser], Unmarshaller]: ...
+def dumps(params: Union[tuple, Fault], methodname: Optional[str] = ..., methodresponse: Optional[bool] = ..., encoding: Optional[str] = ..., allow_none: bool = ...) -> str: ...
+def loads(data: str, use_datetime: bool = ...) -> Tuple[tuple, str]: ...
+
+def gzip_encode(data: str) -> str: ...
+def gzip_decode(data: str, max_decode: int = ...) -> str: ...
+
+class GzipDecodedResponse:
+    stringio = ...  # type: StringIO
+    def __init__(self, response: HTTPResponse) -> None: ...
+    def close(self): ...
+
+class _Method:
+    def __init__(self, send: Callable[[str, tuple], Any], name: str) -> None: ...
+    def __getattr__(self, name: str) -> _Method: ...
+    def __call__(self, *args: tuple) -> Any: ...
+
+class Transport:
+    user_agent = ...  # type: str
+    accept_gzip_encoding = ...  # type: bool
+    encode_threshold = ...  # type: Optional[int]
+    def __init__(self, use_datetime: bool = ...) -> None: ...
+    def request(self, host: _hostDesc, handler, request_body: str, verbose: bool = ...) -> tuple: ...
+    verbose = ...  # type: bool
+    def single_request(self, host: _hostDesc, handler, request_body: str, verbose: bool = ...) -> tuple: ...
+    def getparser(self) -> Tuple[Union[ExpatParser, SlowParser], Unmarshaller]: ...
+    def get_host_info(self, host: _hostDesc) -> Tuple[str, Optional[List[tuple]], Optional[Mapping[Any, Any]]]: ...
+    def make_connection(self, host: _hostDesc) -> HTTPConnection: ...
+    def close(self) -> None: ...
+    def send_request(self, connection: HTTPConnection, handler, request_body: str) -> None: ...
+    def send_host(self, connection: HTTPConnection, host: str) -> None: ...
+    def send_user_agent(self, connection: HTTPConnection) -> None: ...
+    def send_content(self, connection: HTTPConnection, request_body: str) -> None: ...
+    def parse_response(self, response: HTTPResponse) -> tuple: ...
+
+class SafeTransport(Transport):
+    def __init__(self, use_datetime: bool = ..., context: Optional[SSLContext] = ...) -> None: ...
+    def make_connection(self, host: _hostDesc) -> HTTPSConnection: ...
+
+class ServerProxy:
+    def __init__(self, uri: str, transport: Optional[Union[Transport, SafeTransport]] = ..., encoding: Optional[str] = ..., verbose: bool = ..., allow_none: bool = ..., use_datetime: bool = ...) -> None: ...
```

Comment:
I don't think you need the Union since SafeTransport inherits from Transport.

Also, there's another argument called context, which is an `Optional[SSLContext]`.